docs/public-networks/get-started/connect/mainnet.md

```diff
@@ -58,7 +58,7 @@ Run the following command or specify the options in a [configuration file](../..
 
 ```bash
 besu \
-  --sync-mode=X_SNAP           \
+  --sync-mode=SNAP           \
```

Comment:
Hey @macfarla, should we still specify `--sync-mode` and  `--data-storage-format` in the command (since it's now the default).

Reply:
yeah good point - removed